tool/cmd/kitex/main.go

```diff
@@ -45,6 +46,15 @@ func init() {
 			}
 		},
 	})
+	if err := versions.RegisterMinDepVersion(
+		&versions.MinDepVersion{
+			RefPath: "github.com/cloudwego/kitex",
+			Version: "v0.9.0",
+		},
+	); err != nil {
+		log.Warn(err)
+		os.Exit(3)
```

Comment:
Try to avoid magic number: make it a meaningful constant.

Reply:
done.